packages/lib-utils/src/components/details-page-header/utils/ActionMenu.tsx

```diff
@@ -0,0 +1,183 @@
+import type { EitherNotBoth } from '@monorepo/common';
+import {
```

Comment:
I'd really like to avoid having multiple action menus in the SDK if there is overlap between what we are doing in the list view and what we are doing here. I'll take a todo to follow up with @fhlavac